### ISSUE TITLE
adding gerber files

### ISSUE THREAD

**Issue by kartenkarsten** [debaal/SolderingIron#2]
In my opinion working gerber files should be placed here as well.
Maybe in a first hardware release

**Comment by debaal** [comment]
Yes, but the pcb needs to be revised due to a misplacement of an capacitor. The schematic will not be altered though. We can release v1.0 of the pcb after the revision.

**Comment by debaal** [comment]
Yes, but the pcb needs to be revised due to a misplacement of an capacitor. The schematic will not be altered though. We can release v1.0 of the pcb after the revision.

**Comment by oni303** [comment]
see here #4

**Comment by oni303** [comment]
can't see why this issue is closed.
